cmd/server/cmd/server.go

```diff
@@ -20,11 +20,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net"
+	"net/http"
 	"os"
 	"strings"
 	"time"
 
 	bt "github.com/opentracing/basictracer-go"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
I've added it for now, though I am skeptical it provides more detail than the newly-exposed /metrics endpoint.